testcases/OpTestRTCdriver.py

```diff
@@ -115,14 +74,32 @@ def test_RTC_driver(self):
                 continue
         print l_list
 
+
+    ##
+    # @brief This function will cover following test steps
+    #        1. Getting host information(OS and Kernel info)
+    #        2. Loading rtc_opal module based on config option
+    #        3. Testing the rtc driver functions
+    #                Display the current time,
+    #                set the Hardware Clock to a specified time,
+    #                set the Hardware Clock from the System Time, or
+    #                set the System Time from the Hardware Clock
+    #                keep the Hardware clock in UTC or local time format
+    #                Hardware clock compare, predict and adjust functions
+    #                Hardware clock debug and test modes
+    #                Reading the Hardware clock from special file instead of default
+    #        4. After executing above each function reading the Hardware clock in b/w functions.
+    #
+    # @return BMC_CONST.FW_SUCCESS or raise OpTestError
+    #
+    def runTest(self):
+        self.test_init()
```

Comment:
Some thing broken here, When i am running FullRTC test it is just running only test_init function, remaining complete test is not running.

--run testcases.OpTestRTCdriver.FullRTC
test_init (testcases.OpTestRTCdriver.FullRTC) ... OpTestSystem START STATE: 6
OpTestSystem TRANSITIONED TO: 6

hwclock from util-linux 2.27.1
0

4.8.0-32-generic
Config file is available
cat /boot/config-4.8.0-32-generic | grep -i --color=never CONFIG_RTC_DRV_OPAL

CONFIG_RTC_DRV_OPAL=y

Driver built into kernel itself

rtc
rtc0

['/dev/rtc', '/dev/rtc0']
ok

----------------------------------------------------------------------
Ran 1 test in 61.765s

OK


I guess unittest taking method(s) test_* as real test than method runTest when we give both types of test methods.
Can you cross check it once.